bot.user.js

```diff
@@ -314,31 +348,21 @@ var bot = window.bot = (function () {
         sectorBox: {},
         currentFood: {},
         opt: {
-            // target fps
+            // These are the bot's default options
+            // If you wish to customise these, use
+            // customBotOptions above
```

Comment:
Why relocate this to another location? Customization is a good idea, but I don't like the idea of having two separate dicts with settings in them.

Why not just put the defaults into `customBotOptions` and uncomment some of those lines?